database/interface.go

```diff
@@ -18,15 +18,23 @@ import (
 	"github.com/go-vela/server/database/worker"
 )
 
-// Interface represents the interface for Vela integrating
-// with the different supported Database backends.
+// Interface represents the interface for integrating with the supported database providers.
 type Interface interface {
-	// Database Interface Functions
+	// Generic Interface Functions
 
-	// Driver defines a function that outputs
-	// the configured database driver.
+	// TODO: Add this function to the interface once other code has been updated to use the agnostic engine.
+	//
+	// Close defines a function that stops and terminates the connection to the database.
+	// Close() error
```

Comment:
We can't add this function to the interface until we update all code to call `database.NewTest()`.

That will be added in a separate PR because it will touch a lot of code.